plugins/csl-node-editor/src/NodeFactory.cpp

```diff
@@ -19,6 +19,12 @@ void NodeFactory::registerSocketType(std::string name, std::string color) {
 
 ////////////////////////////////////////////////////////////////////////////////////////////////////
 
+void NodeFactory::registerControlType(const std::string& controlSource) {
+  mControls.emplace_back(controlSource);
```

Comment:
I think we usually pass things by value and then move them if they are to be copied.

```suggestion
void NodeFactory::registerControlType(std::string controlSource) {
  mControls.emplace_back(std::move(controlSource));
```